app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<section class="public-recipes-section">
+  <h2 class="pr-header">Public Recipes</h2>
+  <div class="recipe-card-container">
+    <% @recipes.each do |recipe| %>
+      <%= render 'public_recipe_card', recipe: recipe %>
+    <% end %>
+  </div>

```

Comment:
- [ ] Unfortunately, your recipe index page does not match the given wireframe. [Here is the original wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png).
![Screenshot from 2021-12-17 04-30-15](https://user-images.githubusercontent.com/65501535/146473957-5d32b986-56c6-4996-9bfa-59029f67718b.png)